packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -28,38 +29,136 @@ import {HidingColumns} from './HidingColumns';
 import {HidingColumnsAllowsResizing} from './HidingColumnsAllowsResizing';
 import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
 import {Link} from '@react-spectrum/link';
-import {LoadingState, SelectionMode} from '@react-types/shared';
+import {LoadingState} from '@react-types/shared';
 import NoSearchResults from '@spectrum-icons/illustrations/NoSearchResults';
 import {Radio, RadioGroup} from '@react-spectrum/radio';
-import React, {Key, useState} from 'react';
+import React, {Key, useCallback, useState} from 'react';
 import {SearchField} from '@react-spectrum/searchfield';
-import {storiesOf} from '@storybook/react';
 import {Switch} from '@react-spectrum/switch';
 import {TextField} from '@react-spectrum/textfield';
 import {useAsyncList, useListData} from '@react-stately/data';
 import {useFilter} from '@react-aria/i18n';
 import {View} from '@react-spectrum/view';
 
+export default {
+  title: 'TableView',
+  component: TableView,
+  args: {
+    onAction: action('onAction'),
+    onResizeStart: action('onResizeStart'),
+    onResize: action('onResize'),
+    onResizeEnd: action('onResizeEnd'),
+    onSelectionChange: action('onSelectionChange'),
+    onSortChange: action('onSortChange')
+  },
+  argTypes: {
+    // intentionally added so that we can unset the default value
+    // there is no argType for function
+    // use the controls reset button to undo it
+    // https://storybook.js.org/docs/react/essentials/controls#annotation
+    onAction: {
+      control: 'select',
+      options: [undefined]
+    },
```

Comment:
fancy, never thought to try this before

Reply:
it took a while to figure this one out, it's not ideal